App/Modules/SensorConnector/LibreConnection/LibreLinkUpConnection.swift

```diff
@@ -188,7 +188,20 @@ class LibreLinkUpConnection: SensorBluetoothConnection, IsSensor {
         if lastLogin == nil || lastLogin!.authExpires <= Date() {
             DirectLog.info("LibreLinkUp processLogin, starts working, \(Date().debugDescription)")
 
-            let loginResponse = try await login(apiRegion: apiRegion)
+            var loginResponse = try await login(apiRegion: apiRegion)
+            if loginResponse.status == 4 {
```

Comment:
AFAIK a status of 4 means we need to accept terms. I could also type the `step` object so that I could do `loginResponse.data.step.type == "tou"` here, but this works.